aldryn_newsblog/admin.py

```diff
@@ -45,6 +45,7 @@ class ArticleAdminForm(TranslatableModelForm):
 
     class Meta:
         model = models.Article
+        fields = '__all__'
```

Comment:
That's basically just supressing the error message but doesn't fix the actual problem. You should define all fields explictly here.